ocp-config/prov-app/cm.yml

```diff
@@ -131,24 +131,24 @@ objects:
       spring.mail.smtps.connectiontimeout=1000
       spring.mail.properties.mail.smtp.ssl.enable=true
       spring.mail.properties.mail.smtp.auth=true
-      spring.mail.username=${MAIL_USERNAME}
-      spring.mail.password=${MAIL_PASSWORD}
-      provison.mail.sender=provision@${MAIL_HOST}
+      spring.mail.username=${PROV_APP_MAIL_USERNAME}
+      spring.mail.password=${PROV_APP_MAIL_PASSWORD}
+      provison.mail.sender=provision@${PROV_APP_MAIL_HOST}
       spring.main.allow-bean-definition-overriding=true
 parameters:
-- name: JIRA_URI
+- name: PROV_APP_JIRA_URI
   required: true
   description: the URI of JIRA used to create new jira projects  
-- name: CONFLUENCE_URI
+- name: PROV_APP_CONFLUENCE_URI
   required: true
   description: the URI of confluence used to create new confluence spaces
-- name: BITBUCKET_URI
+- name: PROV_APP_BITBUCKET_URI
```

Comment:
@michaelsauter bitbucket URI is used in many placed, or? .. if so - this should not be "prov_app" :)

Reply:
Yes it is. BUT I was not sure if I should use the "globally configured" one here as we have two instances (prov-dev and prov-test) and one might want to configure them differently?

IMHO eventually ODS should ship with one central ProvApp instance in the `cd` namespace (meaning to treat it similar to Nexus/SonarQube). If we were to do that for e.g. v3/v4 then I would propose to just use the globally configured params then. For now though I'd stick with separate configs.

Do you agree or would you rather have a fixed single Atlassian stack for both prov-dev and prov-test instances?

Reply:
I would use the global one ... otherwise - you need to introduce a double pair (dev / test :))

Reply:
I'm with @michaelsauter.

Reply:
ok  then we go with it :)